test/bitcoin_functional/functional/test_framework/util.py

```diff
@@ -302,6 +302,26 @@ def initialize_datadir(dirname, n):
         f.write("discover=0\n")
         f.write("listenonion=0\n")
         f.write("printtoconsole=0\n")
+        # Elements:
```

Comment:
If we're using regtest I don't think these are required... I can do a follow-up PR if true

Reply:
none of them?

Reply:
I'll test later :)